src/Microsoft.Data.SqlClient/netfx/src/Microsoft.Data.SqlClient.csproj

```
@@ -341,7 +341,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Data.SqlClient.SNI">
-      <Version>1.1.0</Version>
+      <Version>1.1.*</Version>
```

Although this may be not a big concern, but I'm thinking of using a range here. 
```suggestion
      <Version>[1.1.0,)</Version>
```
Open for suggestions, which one is better and safer.

---

Comment [here](https://docs.microsoft.com/en-us/nuget/concepts/package-versioning#examples) just got my attention:

>Always specify a version or version range for package dependencies in project files, packages.config files, and .nuspec files. Without a version or version range, NuGet 2.8.x and earlier chooses the latest available package version when resolving a dependency, **whereas NuGet 3.x and later chooses the lowest package version.** Specifying a version or version range avoids this uncertainty.

But testing it locally with NuGet.exe v5+ does not pick smaller version, it does pick largest package version. 🤔

---

Picked up range with upper bound, after internal discussion.